test/lib/ufe/testAttribute.py

```diff
@@ -32,8 +34,12 @@ def __init__(self):
         self._notifications = 0
 
     def __call__(self, notification):
-        if isinstance(notification, ufe.AttributeChanged):
-            self._notifications += 1
+        if(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '2036'):
+            if isinstance(notification, ufe.AttributeValueChanged):
+                self._notifications += 1
+        else:
+            if isinstance(notification, ufe.AttributeChanged):
+                self._notifications += 1
```

Comment:
Oops sorry. Of course the test passed locally for me because I had UFE 0.2.36. I forgot to account for older UFE versions.